uberfire-experimental/uberfire-experimental-client/src/test/java/org/uberfire/experimental/client/workbench/type/ExperimentalAwareClientTypeRegistryImplTest.java

```diff
@@ -30,15 +30,12 @@
 import org.mockito.stubbing.Answer;
 import org.uberfire.client.workbench.type.ClientResourceType;
 import org.uberfire.experimental.client.service.ClientExperimentalFeaturesRegistryService;
-import org.uberfire.experimental.client.workbench.type.test.ClientDiagramResourceType;
-import org.uberfire.experimental.client.workbench.type.test.ClientFormResourceType;
-import org.uberfire.experimental.client.workbench.type.test.ClientJavaResourceType;
-import org.uberfire.experimental.client.workbench.type.test.ClientTextFileResourceType;
-import org.uberfire.experimental.client.workbench.type.test.DiagramResourceType;
-import org.uberfire.experimental.client.workbench.type.test.FormResourceType;
-import org.uberfire.experimental.client.workbench.type.test.JavaResourceType;
-import org.uberfire.experimental.client.workbench.type.test.TextFileResourceType;
-import org.uberfire.experimental.client.workbench.type.test.WrongClientResourceType;
+import org.uberfire.experimental.client.workbench.type.test.client.*;
```

Comment:
In general it is not a good practice to use Star notation in imports it can lead to some bug with hard fix.